python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -1257,7 +1252,9 @@ def do_evaluate(
         right: DataFrame,
     ) -> DataFrame:
         """Evaluate and return a dataframe."""
+        how: str
         how, join_nulls, zlice, suffix, coalesce, _ = options
+        how = how.lower()
```

Comment:
Just curious, now `how` from polars may be uppercase?

Reply:
No that's just in the IR. The `join` API still takes a lowercase literal string.